insights/parsers/nmcli.py

```diff
@@ -153,3 +158,35 @@ def connected_devices(self):
             if 'STATE' in self.data[key] and self.data[key]['STATE'] == 'connected':
                 con_dev.append(key)
         return con_dev
+
+
+@parser(Specs.nmcli_conn_show)
+class NmcliConnShow(CommandParser):
+    """
+    This file will parse the output of all the nmcli connections.
+
+    Sample configuration from a teamed interface in file ``/usr/bin/nmcli conn show``::
+
+       NAME      UUID                                  TYPE      DEVICE
+       enp0s3    320d4923-c410-4b22-b7e9-afc5f794eecc  ethernet  enp0s3
+       virbr0    7c7dec66-4a8c-4b49-834a-889194b3b83c  bridge    virbr0
+       test-net  f858b1cc-d149-4de0-93bc-b1826256847a  ethernet  --
+
+    Examples:
+        >>> type(static_conn)
+        <class 'insights.parsers.nmcli.NmcliConnShow'>
+        >>> static_conn.get_disconnected_connection
+        ['test-net-1']
+    """
+    def parse_content(self, content):
+        self.data = parse_delimited_table(content)
```

Comment:
It will be helpful to explain the attributes `self.data` to an `Attributes` docstring. See https://www.sphinx-doc.org/en/master/usage/extensions/example_google.html

Reply:
acknowledged, updated in the code in latest PR